pyanaconda/payload/dnfpayload.py

```diff
@@ -936,17 +935,6 @@ def environmentId(self, environment):
             raise payload.NoSuchGroup(environment)
         return env.id
 
-    def environmentGroups(self, environmentid, optional=True):
-        env = self._base.comps.environment_by_pattern(environmentid)
-        if env is None:
-            raise payload.NoSuchGroup(environmentid)
-        group_ids = (id_.name for id_ in env.group_ids)
-        option_ids = (id_.name for id_ in env.option_ids)
-        if optional:
-            return list(itertools.chain(group_ids, option_ids))
-        else:
-            return list(group_ids)
-
```

Comment:
@M4rtinK what do you think about this removal?

Reply:
I think this can be safely removed. It likely became dead code when we delegated all group and package exclusion to DNF. Before that we needed to find out what groups are contained in an environment so that we can drop the excluded ones. And apparently we don't need that functionality anywhere else.